rank-info/src/lib.rs

```diff
@@ -0,0 +1,178 @@
+use crate::Message::*;
+use dolphin_integrations::Log;
+use fetcher::RankInfoFetcher;
+use slippi_gg_api::APIClient;
+use slippi_user::*;
+use std::sync::mpsc::{channel, Sender};
+use std::sync::{Arc, Mutex};
+use std::thread;
+
+mod fetcher;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum SlippiRank {
+    Unranked,
+    Bronze1,
+    Bronze2,
+    Bronze3,
+    Silver1,
+    Silver2,
+    Silver3,
+    Gold1,
+    Gold2,
+    Gold3,
+    Platinum1,
+    Platinum2,
+    Platinum3,
+    Diamond1,
+    Diamond2,
+    Diamond3,
+    Master1,
+    Master2,
+    Master3,
+    Grandmaster,
+    Count,
+}
+
+#[derive(Debug, Clone, Default)]
```

Comment:
You can add `#[derive(... Copy ...)]` to this type without issue and avoid needing to call `.clone()` on it.